packages/calcite-components/src/components/list/list.tsx

```diff
@@ -459,6 +516,42 @@ export class List implements InteractiveComponent, LoadableComponent, SortableCo
   //
   // --------------------------------------------------------------------------
 
+  private renderItemAriaLive(): VNode {
+    const {
+      messages,
+      enabledListItems,
+      parentListEl,
+      effectiveLocale,
+      numberingSystem,
+      filterEnabled,
+      filterText,
+      filteredData,
+    } = this;
+
+    numberStringFormatter.numberFormatOptions = {
+      locale: effectiveLocale,
+      numberingSystem,
+    };
+
+    const count = numberStringFormatter.localize(enabledListItems.length.toString());
+
+    const showFilteredExt = filterEnabled && filterText && filteredData?.length;
+
+    return !parentListEl ? (
+      <div aria-live="polite" class={CSS.assistiveText}>
+        {showFilteredExt ? <div key="aria-filter-enabled">{messages.filterEnabled}</div> : null}
+        <div key="aria-item-count">{messages.total.replace("{count}", count)}</div>
```

Comment:
Maybe we should have or use a substitution utility for substituting message text? Instead of just using string replace? cc @jcfranco.